addons/web/static/src/views/fields/parsers.js

```diff
@@ -33,13 +33,13 @@ function evaluateMathematicalExpression(expr, context = {}) {
  *
  * @param {string} value
  * @param {Object} options - additional options
- * @param {string|RegExp} [options.thousandsSep] - the thousands separator used in the value
- * @param {string|RegExp} [options.decimalPoint] - the decimal point used in the value
+ * @param {string|RegExp|bool} options.thousandsSep - the thousands separator used in the value
+ * @param {string|RegExp} options.decimalPoint - the decimal point used in the value
  * @returns {number}
  */
 function parseNumber(value, options = {}) {
     // a number can have the thousand separator multiple times. ex: 1,000,000.00
-    value = value.replaceAll(new RegExp(escapeRegExp(options.thousandsSep), "g") || ",", "");
+    value = value.replaceAll(new RegExp(escapeRegExp(options.thousandsSep || ""), "g"), "");
```

Comment:
I meant, the fallback should be done in the method that calls parseNumber with a possibly unset thousandsSep (i.e. parseFloat)

Reply:
got it now sorry